components/datagrid/HeaderRowOverlay/NumSelectedRows.js

```diff
@@ -0,0 +1,42 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import Box from '../../box';
+import { TextBody, Monospaced } from '../../typography';
+import theme from './theme.css';
+
+class NumSelectedRows extends PureComponent {
+  render() {
+    const { numSelectedRows, numSelectedRowsLabel } = this.props;
+
+    return (
+      <Box
+        display="flex"
+        align-items="center"
```

Comment:
I don't think this works with a dash in TL-UI. Should be camelCased, alignItems, right?